evap/staff/templates/staff_course_email.html

```diff
@@ -17,13 +17,15 @@
                     <div class="form-row">
                         {% if field.name == "recipients" %}
                             <div class="form-label vertically-aligned {% if field.errors %}error{% endif %}" name="{{ field.name }}">{{ field.label }}</div>
-                            <div class="form-field btn-group" data-toggle="buttons">
+                            <div class="btn-group form-field">
+                            <div class="btn-group" data-toggle="buttons">
                                 {% for choice in field %}
                                     <label class="btn btn-sm btn-default {% if choice.choice_value in field.value %}active{% endif %} {% if field.errors %}error{% endif %}" name="{{ field.name }}" onclick="changedFieldValue('{{ field.name }}')">
                                         <input id="{{ choice.id_for_label }}" name="{{ choice.name }}" type="checkbox" value="{{ choice.choice_value }}" autocomplete="off" {% if choice.choice_value in field.value %}checked{% endif %} />
                                         {{ choice.choice_label }} {{ choice.id }}
                                     </label>
                                 {% endfor %}
+                            </div><input name="export" type="submit" value="{% trans "Export" %}" class="btn btn-sm btn-primary"/>
```

Comment:
can you please move the button behind the `btn-group`? and please call it `Show recipients`